spec/requests/registrations_spec.rb

```diff
@@ -137,10 +137,11 @@
         allow(FeatureFlag).to receive(:enabled?).with(:creator_onboarding).and_return(true)
         allow(FeatureFlag).to receive(:enabled?).with(:runtime_banner).and_return(false)
         allow(SiteConfig).to receive(:waiting_on_first_user).and_return(true)
+        allow(Settings::UserExperience).to receive(:public).and_return(false)
```

Comment:
makes sense - we have this setup in the e2e seed (to public => true), but aren't testing this first user experience there.